plugins/modules/win_route.py

```diff
@@ -11,16 +12,21 @@
 description:
     - Add or remove a static route.
 options:
-  destination:
+  network:
     description:
-      - Destination IP address in CIDR format (ip address/prefix length).
+      - Destination network.
+    type: str
+    required: yes
+  mask:
+    description:
+      - Destination network's mask.
     type: str
     required: yes
   gateway:
     description:
         - The gateway used by the static route.
-        - If C(gateway) is not provided it will be set to C(0.0.0.0).
     type: str
+    required: yes
```

Comment:
This will break existing users who don't set `gateway` and rely on the defaults.